prody2/viewers/viewer_gnm.py

```diff
@@ -0,0 +1,266 @@
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+"""
+This module implement the wrappers around ProDy GNM 
+visualization programs.
+"""
+
+from pyworkflow.gui.project import ProjectWindow
+from pyworkflow.protocol.params import LabelParam, IntParam, BooleanParam
+from pyworkflow.viewer import ProtocolViewer, DESKTOP_TKINTER, WEB_DJANGO
+
+from pwem.viewers.plotter import EmPlotter
+
+from pwem.viewers import ObjectView, VmdView, DataView
+from pwem.emlib import MDL_NMA_ATOMSHIFT
+from pwem.objects import SetOfNormalModes
+
+from continuousflex.protocols import FlexProtNMA
+from continuousflex.viewers.nma_plotter import FlexNmaPlotter
+
+from prody2.protocols import ProDyGNM
+
+import os
+
+import prody
+from prody.utilities.drawtools import IndexFormatter
+from matplotlib.pyplot import *
+
+OBJCMD_NMA_PLOTDIST = "Plot distance profile"
+OBJCMD_NMA_VMD = "Display VMD animation"
+
+
+class ProDyGNMViewer(ProtocolViewer):
+    """ Visualization of results from the GNM-NMA protocol.    
+        Normally, modes with high collectivity and low 
+        score are preferred.
+    """
+    _label = 'viewer gnm'
+    _targets = [ProDyGNM]
+    _environments = [DESKTOP_TKINTER, WEB_DJANGO]
+
+#     def setProtocol(self, protocol):
+#         ProtocolViewer.setProtocol(self, protocol)
+#         inputPdb = protocol.inputStructure.get()
+#         self.isEm.set(inputPdb.getPseudoAtoms())
+        
+    def _defineParams(self, form):
+
+        if isinstance(self.protocol, SetOfNormalModes):
+            protocol_path = os.path.dirname(os.path.dirname(self.protocol[1].getModeFile()))
+            vmdFiles = protocol_path + "/extra/animations/"
+            nmdFile = protocol_path + "/modes.nmd"
+        else:
+            vmdFiles = self.protocol._getExtraPath("animations")
+            nmdFile = self.protocol._getPath("modes.nmd")
+
+        form.addSection(label='Visualization')
+  
+        form.addParam('displayModes', LabelParam,
+                      label="Display output Normal Modes?", important=True)
+        form.addParam('displayMaxDistanceProfile', LabelParam,
+                      label="Plot max distance profile?",
+                      help="Maximum unitary shift of each atom or pseudoatom over all computed modes.") 
+
+        form.addParam('displaySqFlucts', LabelParam,
+                      label="Plot MSF from all the computed modes?",
```

Comment:
would be good to define MSF as mean square fluctuations and put MSF in brackets. Perhaps in the help